image_opts.go

```diff
@@ -0,0 +1,47 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package containerd
+
+import (
+	encconfig "github.com/containerd/containerd/pkg/encryption/config"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// LayerToolDecryptData holds data that the external layer decryption tool
```

Comment:
This should be moved the the encryption pkg

Reply:
We have `pkg/encryption`, which would be non-containerd specific and ideally be from an OCI github repo. So I would put it into `images/encryption/`. Any specific file and/or directory?

Reply:
@dmcgowan any ideas?